src/main/resources/view/CustomTheme.css

```diff
@@ -0,0 +1,352 @@
+  -fx-text-fill: #555555;
+  -fx-opacity: 0.9;
+}
+
+.label-bright {
+  -fx-font-size: 11pt;
+  -fx-font-family: "Segoe UI Semibold";
+  -fx-text-fill: white;
+  -fx-opacity: 1;
+}
+
+.label-header {
+  -fx-font-size: 32pt;
+  -fx-font-family: "Segoe UI Light";
+  -fx-text-fill: white;
+  -fx-opacity: 1;
+}
+
+.text-field {
+  -fx-font-size: 12pt;
+  -fx-font-family: "Segoe UI Semibold";
+}
+
+.tab-pane {
+  -fx-padding: 0 0 0 1;
+}
+
+.tab-pane .tab-header-area {
+  -fx-padding: 0 0 0 0;
+  -fx-min-height: 0;
+  -fx-max-height: 0;
+}
+
+.table-view {
+  -fx-base: #1d1d1d;
+  -fx-control-inner-background: #1d1d1d;
+  -fx-background-color: #1d1d1d;
+  -fx-table-cell-border-color: transparent;
+  -fx-table-header-border-color: transparent;
+  -fx-padding: 5;
+}
+
+.table-view .column-header-background {
+  -fx-background-color: transparent;
+}
+
+.table-view .column-header, .table-view .filler {
+  -fx-size: 35;
+  -fx-border-width: 0 0 1 0;
+  -fx-background-color: transparent;
+  -fx-border-color:
+      transparent
+      transparent
+      derive(-fx-base, 80%)
+      transparent;
+  -fx-border-insets: 0 10 1 0;
+}
+
+.table-view .column-header .label {
+  -fx-font-size: 20pt;
+  -fx-font-family: "Segoe UI Light";
+  -fx-text-fill: white;
+  -fx-alignment: center-left;
+  -fx-opacity: 1;
+}
+
+.table-view:focused .table-row-cell:filled:focused:selected {
+  -fx-background-color: -fx-focus-color;
+}
+
+.split-pane:horizontal .split-pane-divider {
+  -fx-background-color: derive(#1d1d1d, 20%);
+  -fx-border-color: transparent transparent transparent #4d4d4d;
+}
+
+.split-pane {
+  -fx-border-radius: 1;
+  -fx-border-width: 1;
+  -fx-background-color: derive(#1d1d1d, 20%);
+}
+
+.list-view {
+  -fx-background-insets: 0;
+  -fx-padding: 0;
+  -fx-background-color: derive(#1d1d1d, 20%);
+}
+
+.list-cell {
+  -fx-label-padding: 0 0 0 0;
+  -fx-graphic-text-gap : 0;
+  -fx-padding: 0 0 0 0;
+}
+
+.list-cell:filled:even {
+  -fx-background-color: #3c3e3f;
+}
+
+.list-cell:filled:odd {
+  -fx-background-color: #515658;
+}
+
+.list-cell:filled:selected {
+  -fx-background-color: #424d5f;
+}
+
+.list-cell:filled:selected #cardPane {
+  -fx-border-color: #3e7b91;
+  -fx-border-width: 1;
+}
+
+.list-cell .label {
+  -fx-text-fill: white;
+}
+
+.cell_big_label {
+  -fx-font-family: "Segoe UI Semibold";
+  -fx-font-size: 16px;
+  -fx-text-fill: #010504;
+}
+
+.cell_small_label {
+  -fx-font-family: "Segoe UI";
+  -fx-font-size: 13px;
+  -fx-text-fill: #010504;
+}
+
+.stack-pane {
+  -fx-background-color: derive(#1d1d1d, 20%);
+}
+
+.pane-with-border {
+  -fx-background-color: derive(#1d1d1d, 20%);
+  -fx-border-color: derive(#1d1d1d, 10%);
+  -fx-border-top-width: 1px;
+}
+
+.status-bar {
+  -fx-background-color: derive(#1d1d1d, 30%);
+}
+
+.result-display {
+  -fx-background-color: transparent;
+  -fx-font-family: "Segoe UI Light";
+  -fx-font-size: 13pt;
+  -fx-text-fill: white;
+}
+
+.result-display .label {
+  -fx-text-fill: black !important;
```

Comment:
Why codacy complaining this?

Reply:
The code is adapted from darktheme.css which is given by AddressBook 3

Reply:
This is not an issue, can be merged